src/main/java/de/tum/in/www1/artemis/web/rest/TextAssessmentResource.java

```diff
@@ -83,7 +90,27 @@ public TextAssessmentResource(AuthorizationCheckService authCheckService,
         }
 
         Participation participation = textSubmission.get().getParticipation();
-        participation  = participationService.findOneWithEagerResultsAndSubmissions(participation.getId());
+        participation = participationService.findOneWithEagerResultsAndSubmissions(participation.getId());
```

Comment:
Please add JavaDoc to this method and specify in detail what you are doing here. Otherwise the code is not really understandable

Reply:
Done